### PR TITLE
Change compiler from dpcpp to icpx -fsycl Part 1

### PR DESCRIPTION
**Pull request by KanclerzPiotr**
# Existing Sample Changes
## Description

First Part of compiler change from dpcpp to icpx -fsycl. This

Fixes Issue# 

## External Dependencies

List any external dependencies created as a result of this change.

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X] Implement fixes for ONSAM Jiras

- [X] Command Line

### CODE REVIEW

File: DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp [hunk 1]
```diff
@@ -63,19 +63,19 @@ int main() {
   // Create device selector for the device of your interest.
 #if FPGA_EMULATOR
   // DPC++ extension: FPGA emulator selector on systems without FPGA card.
```
Comment (jimmytwei) [inline DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp:65]: @KanclerzPiotr Do you want to remove the phrase "DPC++ extension" from the comment?

File: DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp [hunk 2]
```diff
@@ -63,19 +63,19 @@ int main() {
...
-  ext::intel::fpga_emulator_selector d_selector;
+  const auto& selector = ext::intel::fpga_emulator_selector_v;
 #elif FPGA
   // DPC++ extension: FPGA selector on systems with FPGA card.
```
Comment (jimmytwei) [inline DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp:68]: @KanclerzPiotr Do you want to remove the phrase "DPC++ extension" from the comment?

File: DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp (hunk 1 above)
Comment (KanclerzPiotr) [reply DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp:65]: I think we should use Intel or oneAPI extension. This is not part of the specification.

File: DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp (hunk 2 above)
Comment (KanclerzPiotr) [reply DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp:68]: I think we should use Intel or oneAPI extension. This is not part of the specification.

File: DirectProgramming/DPC++/ParallelPatterns/loop-unroll/src/loop-unroll.cpp [hunk 3]
```diff
@@ -94,7 +94,7 @@ int main() {
   vector<float> sum(n);
 
   try {
-    queue q(default_selector{}, dpc_common::exception_handler,
```
Comment (praveenkk123) [inline DirectProgramming/DPC++/ParallelPatterns/loop-unroll/src/loop-unroll.cpp:97]: Piotr,
You removed exception_handler from some of the code snippets but not from the other snippets like here?.

**Review by pmpeter1 — DISMISSED** [review]
The C++ changes look good.

File: DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp (hunk 1 above)
Comment (KanclerzPiotr) [reply DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp]: I fixed this with rename to Intel extension

File: DirectProgramming/DPC++/ParallelPatterns/loop-unroll/src/loop-unroll.cpp (hunk 3 above)
Comment (KanclerzPiotr) [reply DirectProgramming/DPC++/ParallelPatterns/loop-unroll/src/loop-unroll.cpp:97]: I removed all the handlers

**Review by jimmytwei — DISMISSED** [review]

**Review by jimmytwei — APPROVED** [review]

**Comment by jimmytwei** [comment]
@KanclerzPiotr Can you remove Makefile and Makefile.fpga from the simple-add folder and resubmit the PR? Yohann removed all Makefiles and cleaned everything up so that we have just one CMakeLists.txt for simple-add and vector-add.

**Review by jimmytwei — APPROVED** [review]

**Pull request merged**